### ISSUE TITLE
Question about 

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9671]
1. What promotion does **MULTIPACK** and **CROSSMULTIPACK**  in **/order/checkout-forms/{id}** represent?
2.If a customer used a **gift card** or **coins** when purchasing an item, which field tells us whether the order used this promotions and what the value was？
3. Is there a formula for calculating the order amount? Is this calculation formula correct?**summary.totalToPay.amount=payment.paidAmount.amount=delivery.cost.amount+lineitem.price.amount*lineitem.quantity+surcharges.paidamount.amount**

**Comment by yangcen6** [comment]
But for Q2, we need this information to calculate what specific discounts are available for this order.

**Comment by MaciejFrackowiak** [comment]
Gift cards or coins are not the discounts, but some kind of payment method and such information is not necessary for the seller because he receives the whole order amount.

**Comment by yangcen6** [comment]
There is no corresponding field in the payment method. For example, coins are borne by the merchant. We need to know the specific amount of this part for accounting and invoicing.

**Comment by MaciejFrackowiak** [comment]
Coins are given by Allegro to the buyers too. You can always track costs using [GET /billing/billing-entries](https://developer.allegro.pl/documentation/#operation/getBillingEntries).

**Comment by yangcen6** [comment]
For Q3，Is this calculation formula correct？ lineitem.price.amount=lineitem.originalprice.amount-lineitem.vouchers.value.amount-discounts.value.amount

**Comment by MaciejFrackowiak** [comment]
Yes, lineItem.price.amount is a value after any discount.

**Comment by yangcen6** [comment]
Is the order payment time returned by the API Polish time?

**Comment by MaciejFrackowiak** [comment]
No, in the API we return universal time, Zulu Time. Please, read [basic](https://developer.allegro.pl/tutorials/basic-information-VL6YelvVKTn) information about Allegro API.
